### PR TITLE
Updated ionize to look for multiple external test entries

### PR DESCRIPTION
**Pull request by ddanielr**

### CODE REVIEW

File: cmd/analysis.go [hunk 1]
```diff
@@ -58,16 +58,24 @@ Will read the configuration from the $PWD/.ionize.yaml file and begin an analysi
...
 			}
-
-			fmt.Println("Adding external vulnerabilities scan data")
-			analysisStatus, err = cli.AddScanResult(id, team, project, "accepted", "vulnerability", *scan)
-			if err != nil {
-				log.Fatalf("Analysis Report request failed for %s: %v", project, err.Error())
+			for _, file := range files {
+				scan, err := loadVulnerabilities(file)
+				if err != nil {
+					log.Fatalf("Analysis request failed for %s: %v", project, err.Error())
```
Comment (dan9186) [inline cmd/analysis.go]: does fatal mean that it should do an `os.Exit(1)` or something like that?

File: cmd/analysis.go [hunk 2]
```diff
@@ -58,16 +58,24 @@ Will read the configuration from the $PWD/.ionize.yaml file and begin an analysi
...
+				}
+
+				fmt.Println("Adding external vulnerabilities scan data")
```
Comment (dan9186) [inline cmd/analysis.go]: Any reason to not use the logger for this one?

File: cmd/analysis.go (hunk 1 above)
Comment (ddanielr) [reply cmd/analysis.go]: yes.
// Fatalf is equivalent to Printf() followed by a call to os.Exit(1).
func Fatalf(format string, v ...interface{}) {
	std.Output(2, fmt.Sprintf(format, v...))
	os.Exit(1)
}

File: cmd/analysis.go (hunk 2 above)
Comment (ddanielr) [reply cmd/analysis.go]: we use fmt.Println for all stdout statements. the logger tends to output timedate stuff which makes the output a bit odd. Throughout the file we are being consistent, but let's have a discussion on the best way forward for this later.

**Pull request merged**